media_kit_test/lib/tests/01.single_player_single_video.dart

```diff
@@ -64,6 +64,7 @@ class _SinglePlayerSingleVideoScreenState
             ),
             onTap: () {
               player.open(Media('asset://assets/video_$i.mp4'));
+              player.play();
```

Comment:
Why?

Reply:
It's not right to make redundant changes to the test application. This test application is to test the rendering etc.
The playback, playlist, event streams etc. should be tested as part of `media_kit/*` or `media_kit/tests/*`.

Reply:
> It's not right to make redundant changes to the test application. This test application is to test the rendering etc.
> The playback, playlist, event streams etc. should be tested as part of `media_kit/*` or `media_kit/tests/*`.

Ok, so you want me to remove it, and add it to media_kit test right?